coral-schema/src/main/java/com/linkedin/coral/schema/avro/ViewToAvroSchemaConverter.java

```diff
@@ -196,6 +196,9 @@ private Schema inferAvroSchema(String dbName, String tableOrViewName, boolean st
     } else {
       RelNode relNode = hiveToRelConverter.convertView(dbName, tableOrViewName);
       Schema schema = relToAvroSchemaConverter.convert(relNode, strictMode, forceLowercase);
+      if (forceLowercase) {
```

Comment:
is it a bit confusing that for views, we force LowerCase at this point, but for tables, that transformation is inside schemaUtilities?
For consistency and readability, can we visit `ToLowercaseSchemaVisitor` for both tables and views in this layer when `forceLowercase` is true? then probably `schemaUtilites` doesn't need to worry about lowercase flag?

Reply:
thanks, I have removed the flag `forceLowercase` for schema utility as well, lowercase is handed in  [ViewToAvroSchemaConverter.java](https://github.com/linkedin/coral/pull/275/files/818f6ce19615b6221d875a2474a8c96dc1f34003#diff-29d8b6a6932537360d80a64e0114c5da3e86b43a3f7f30c3e2b70cfc22dc581b) for both tables and views